bluesky/plan_patterns.py

```diff
@@ -229,10 +239,15 @@ def spiral_fermat(x_motor, y_motor, x_start, y_start, x_range, y_range, dr,
     -------
     cyc : cycler
     '''
+    if dr_y is None:
+        dr_aspect = 1
+    else:
+        dr_aspect=dr_y/dr
```

Comment:
Blanks spaces? Or some people may better like a one-liner, e.g.:
```py
dr_aspect = 1 if not dr_y else dr_y / dr
```

Reply:
Please don't write code like that - it's obfuscated.  Stick to the indented form.  It is very clear.

Reply:
Maybe. Either way is fine to me. List comprehensions can be unclear too, but we use them a lot, e.g. in https://github.com/NSLS-II/bluesky/blob/master/bluesky/callbacks/fitting.py. Just a thought. I'm +1 for clearness!

Reply:
It's a fair point. As a data point, I think this specific case is against Google code style (not that we strictly follow that or anything) whereas list comprehensions are fine.